file/delete_scheduler.h

```diff
@@ -91,6 +92,8 @@ class DeleteScheduler {
 
   void BackgroundEmptyTrash();
 
+  void MaybeCreateBackgroundThread();
```

Comment:
Sorry for this last minute comment. Adding a one-liner comment for this function will be good.